common-test/src/main/scala/org/genivi/sota/data/PackageIdGenerators.scala

```diff
@@ -22,6 +22,17 @@ trait PackageIdGenerators {
   val genPackageIdVersion: Gen[PackageId.Version] =
     Gen.listOfN(3, Gen.choose(0, 999)).map(_.mkString(".")).map(Refined.unsafeApply) // scalastyle:ignore magic.number
 
+  def genConflictFreePackageIdVersion(n: Int): Seq[PackageId.Version] = {
```

Comment:
We could generalise this to more generators, by having this take a `Gent[T]`, or do some `implicit class` like in `GeneratorOps`.